app/backend/approaches/readdecomposeask.py

```diff
@@ -42,7 +42,7 @@ def search(self, q: str, overrides: dict) -> str:
             self.results = [doc[self.sourcepage_field] + ":" + nonewlines(doc[self.content_field][:500]) for doc in r]
         return "\n".join(self.results)
 
-    def lookup(self, q: str) -> str:
+    def lookup(self, q: str) -> Optional[str]:
```

Comment:
Since the last line of this method returns `None`, `str` is not a valid annotation as it can return None

Reply:
Since you changed this, employee_info() should have same return type?

Reply:
that's being addressed in another PR